models/webhooks/store_option_updated.yml

```diff
@@ -0,0 +1,32 @@
+type: object
+properties:
+  store/option/updated:
+    description: Fires when product options are edited. 
+    type: object
+    properties:
+      scope:
+        type: string
+      store_id:
+        type: string
+      data:
+        type: object
+        properties:
+          type:
+            type: string
+          id:
+            type: integer
+          affected_product_ids: 
```

Comment:
could you also mention that we have a limit for this, so we return maximum 100 product ids in array?